cinema_service/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("api/user/", include("user.urls", namespace="user")),
     path("__debug__/", include("debug_toolbar.urls")),
```

Comment:
The `debug_toolbar` URL pattern is included. Ensure it is only enabled in development environments, not in production, to maintain security.